### PR TITLE
Dansby/mobile modal

### PR DESCRIPTION
**Pull request by bendansby**
Some tweaks for modals on mobile devices.

IMG_2896
IMG_2897

### CODE REVIEW

File: src/Nri/Ui/Modal/V11.elm [hunk 1]
```diff
@@ -425,16 +426,20 @@ modalStyles =
...
     , borderRadius (px 20)
     , boxShadow5 zero (px 1) (px 10) zero (rgba 0 0 0 0.35)
+    , Css.Media.withMedia
+        [ Css.Media.all [ Css.Media.maxWidth (px 1000) ] ]
+        [ borderRadius zero
+        ]
 
     -- Spacing
-    , margin2 (px 50) auto
+    , margin2 (px 20) auto
```
Comment (bendansby) [inline src/Nri/Ui/Modal/V11.elm:435]: This is intended for both mobile and desktop.

File: src/Nri/Ui/Modal/V11.elm [hunk 2]
```diff
@@ -425,16 +426,20 @@ modalStyles =
...
 
     -- Size
     , width (px 600)
     , backgroundColor Colors.white
 
     -- the modal should grow up to the viewport minus a 50px margin
```
Comment (tesk9) [inline src/Nri/Ui/Modal/V11.elm:441]: Maybe we should update this comment too?

**Review by tesk9 — COMMENTED** [review]
Looks good! The chrome mobile previewer was not very helpful but I looked at it on my actual phone (thanks, netlify previews!) and it's definitely a big improvement!

One thing on the margin2 mobile & desktop change -- I think this allows for some content overlap:

### Before
Screen Shot 2021-09-29 at 11 40 45 AM

### On this PR
Screen Shot 2021-09-29 at 11 40 20 AM

File: src/Nri/Ui/Modal/V11.elm (hunk 2 above)
Comment (bendansby) [reply src/Nri/Ui/Modal/V11.elm]: Good catch, updated.

**Comment by bendansby** [comment]
Thanks! I think I'm OK with that overlap. Will see how it feels once it's in the site and make adjustments as necessary.

**Pull request merged**